src/lib/draw-and-pick.js

```diff
@@ -40,7 +40,7 @@ export function drawLayers({layers, pass}) {
           viewport: layer.context.viewport
         }),
         uniforms: Object.assign(
-          {renderPickingBuffer: 0, pickingEnabled: 0},
+          {renderPickingBuffer: 0, pickingEnabled: 0, picking_uActive: 0},
```

Comment:
Suggest splitting this into three lines and commenting on each of these how/where it is used.